### DISCUSSION TITLE
stitches example reset is an empty function?

### DISCUSSION THREAD

**Discussion by gutenye** [remix-run/remix#3129] [Q&A] [answered]
Hey love the remix framework, I have a question about integrating the stitches. The reset function is empty, how does an empty function work?

https://github.com/remix-run/remix/blob/b1d72b76cc52b09fb84a7d8258fced7ea08c6477/examples/stitches/app/styles/client.context.ts#L9

**Comment by janhoogeveen** [comment, marked as answer]
It's a no-op function to set-up the context provider. Probably needed to satisfy TypeScript. If you look into [entry.client.tsx](https://github.com/remix-run/remix/blob/b1d72b76cc52b09fb84a7d8258fced7ea08c6477/examples/stitches/app/entry.client.tsx#L19) you can see the no-op gets overwritten by a functional reset implementation

**Reply by gutenye** [reply]
Thanks
